index.html

```diff
@@ -970,7 +970,10 @@ <h1>
 		<div class="span3" title="Code: 0xe9bd"><i class="demo-icon eicon-nested-carousel">&#xe9bd;</i> <span class="i-name">eicon-nested-carousel</span><span class="i-code">0xe9bd</span></div>
 		<div class="span3" title="Code: 0xe9be"><i class="demo-icon eicon-ai">&#xe9be;</i> <span class="i-name">eicon-ai</span><span class="i-code">0xe9be</span></div>
 		<div class="span3" title="Code: 0xeb7d"><i class="demo-icon eicon-taxonomy-filter">&#xeb7d;</i> <span class="i-name">eicon-taxonomy-filter</span><span class="i-code">0xeb7d</span></div>
+	</div>
+	<div class="row">
 		<div class="span3" title="Code: 0xef02"><i class="demo-icon eicon-container-grid">&#xef02;</i> <span class="i-name">eicon-container-grid</span><span class="i-code">0xef02</span></div>
+		<div class="span3" title="Code: 0xeb82"><i class="demo-icon eicon-upgrade">&#xeb82;</i> <span class="i-name">eicon-upgrade</span><span class="i-code">0xeb82</span></div>
 </div>
```

Comment:
@hein-obox Are you missing a closing div tag here?

Reply:
@hein-obox 
of the new row